### PR TITLE
Server fix

### PR DESCRIPTION
**Pull request by RSiferd2**
Got routing to sort of work, there's still a lot that needs to be done though...

### CODE REVIEW

File: client/js/app.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ angular.module('fridgely', [
   'fridgely.recipes',
   'fridgely.landing',
   'fridegly.search',
-  'ngRoute'
```
Comment (RSiferd2) [inline client/js/app.js]: This fixes an error pomander was giving

File: client/js/app.js [hunk 2]
```diff
@@ -4,8 +4,8 @@ angular.module('fridgely', [
   'fridgely.recipes',
   'fridgely.landing',
   'fridegly.search',
```
Comment (RSiferd2) [inline client/js/app.js]: This fixes an error pomander was giving.

File: server/server.js [hunk 3]
```diff
@@ -24,24 +25,24 @@ req.body has a property with an array of ingredients:
 {
   "ingredients": [ "hotdogs", "flour", "milk" ]
 }
-
+g
```
Comment (brianzhou13) [inline server/server.js]: I have some confusion behind this 'g'? is this a typo?

Comment (RSiferd2) [inline server/server.js]: Yes... fixing

**Comment by brianzhou13** [comment]
looks good. 

Resolve #45

**Pull request merged**